### ISSUE TITLE
Nvda can not read the paypal page for donation

### ISSUE THREAD

**Issue by jeanbrazcosta** [nvaccess/nvda#8018]
Nvda can not read the paypal page for donation

reproduction

1: I visited the nvda page
https://www.nvaccess.org/donate/
2: I marked the other option, and I typed 5
and  clicked to donate
3: being on the paypal page at
https://www.paypal.com/donate/?token=NXE3MoBDZuImKOCA6s4oZBbY-VcjE5XCXFU8WH7hJ6AWcZBhZ6SF93sbWKYr2UbA3CrY1G&country.x=US&locale.x=
nvda does not read
using tab, arrows can not complete operation

Nvda Instaled version: 2017.4

Windows 10
browser: chrome 64.0.3282.167  64 bits
[log nvda.txt](https://github.com/nvaccess/nvda/files/1742122/log.nvda.txt)

**Comment by ehollig** [comment]
Are you using Chrome version 64.0.3282.167? I was able to reproduce this once using Chrome 63, but after the update to Chrome 64, it seems to be working correctly.

**Comment by jeanbrazcosta** [comment]
I reestarded My computer but I still has a problem

2018-02-20 19:58 GMT-03:00 Ethan Holliger <notifications@github.com>:

**Comment by jeanbrazcosta** [comment]
I ran the test on another computer and it worked correctly.
